src/main/resources/templates/subscription.html

```diff
@@ -264,8 +264,8 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                             <label class="h5 pl-1 control-label font-weight-bold">Authorization</label>
                             <img class="cursor-pointer" id="authInfo" alt="Auth Info"
                                 src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
-                                title="This selector enables you to select differen authentication methods. <br>
-                                    NO_AUTH: No Authorization used<br>
+                                title="This selector enables you to select different authorization methods. <br>
```

Comment:
Here we are using both authentication and authorization, they are two different things(I don't know if BASIC_AUTH is doing both or not though). Either write auth(which relates to both) or authentication/authorization.

Reply:
Done